### PR TITLE
add: Terms of Service and Cookie Policy

### PR DESCRIPTION
**Pull request by Dayitva**

### CODE REVIEW

File: socialite/src/components/Policies/Policy.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.container{
```
Comment (prajneya) [inline socialite/src/components/Policies/Policy.css]: container is a reserved bootstrap class. Might also affect other components, rename this to something else like policy-container

File: socialite/src/components/Policies/Terms.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
+import React, { useContext } from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+import "./Policy.css"
+
+function Terms(props){
+
+	return (
+          <>
+          		<div className="container">
```
Comment (prajneya) [inline socialite/src/components/Policies/Terms.js]: policy-container

File: socialite/src/components/Register/Register.js [hunk 3]
```diff
@@ -183,6 +183,10 @@ function Register(props) {
 
 							<button type="submit" className="btn-submit">Sign Up</button>
 
+							<div>
+							By clicking Sign Up, you agree to our <a href="https://www.peersity.in/policies/terms">Terms</a> and <a href ="https://www.peersity.in/policies/cookies">Cookie Policy</a>.
+							</div>
+
```
Comment (prajneya) [inline socialite/src/components/Register/Register.js:189]: has this been tested to be responsive?

Comment (Dayitva) [reply socialite/src/components/Register/Register.js:189]: Done

File: socialite/src/components/Policies/Terms.js (hunk 2 above)
Comment (Dayitva) [reply socialite/src/components/Policies/Terms.js]: Done

**Pull request merged**